packages/eth2.0-state-transition/src/block/operations/index.ts

```diff
@@ -67,7 +68,7 @@ export function processOperations(
     func,
     verifySignatures,
   }: {
-    operations: Operation[];
+    operations: List<Operation>;
```

Comment:
Is that necessary? Operation isn't ssz type

util functions?

Reply:
Yeah, its necessary. If we accept that ssz objects can have different backings, we can only support the lowest-common-denominator interface between them.
This is what is currently defined in `ssz/src/interface.ts`.
The idea is we want to be able to seemlessly swap out a structural beacon state with a tree-backed beacon state.

Reply:
I get that, but this isn't part of state and Operation type is not ssz type